pkg/errors/clusters.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+import (
+	"fmt"
+
+	commonerrors "sigs.k8s.io/cluster-api/pkg/apis/cluster/common"
```

Comment:
I'm curious why you chose to rename this import when there wasn't a conflict.

Reply:
I wanted it to be consistent with `pkg/errors/machines.go`, it's also renamed there.